modules/storage-api/src/main/java/org/apache/ignite/internal/storage/engine/MvTableStorage.java

```diff
@@ -36,7 +36,7 @@
 /**
  * Table storage that contains meta, partitions and SQL indexes.
  */
-public interface MvTableStorage {
+public interface MvTableStorage extends AutoCloseable {
```

Comment:
Can method `close()` be defined explicitly?
This will get rid of warnings on the idea that we don't close the resource after it's been used.
@sanpwc wdyt

Reply:
Fixed.